gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/datafetcher/QueryServiceEntityFetcherTests.java

```diff
@@ -191,7 +193,6 @@ public void test_getTotalEntitiesSingleEntityIdAttribute() {
             .setEntityType(entityType.name())
             .setStartTimeMillis(startTime)
             .setEndTimeMillis(endTime)
-            .addSelection(buildExpression(API_NAME_ATTR))
```

Comment:
Why was this removed?

Reply:
Hmm.. was trying to modify this test case to cover both use cases but not needed so reverted it.